### PR TITLE
feat(editor3): Remove all formatting button

### PR DESCRIPTION
**Pull request by pablopunk**
SDANSA-422

Pressing the button will convert all the text of the editor in plain
text (even removing line breaks).

### CODE REVIEW

File: scripts/apps/workspace/content/directives/ContentProfileSchemaEditor.ts [hunk 1]
```diff
@@ -93,6 +91,7 @@ const EDITOR3_RICH_FORMATTING_OPTIONS = [
     'table',
     'formatting marks',
     'remove format',
+    'remove all format',
```
Comment (tomaskikutis) [inline scripts/apps/workspace/content/directives/ContentProfileSchemaEditor.ts:94]: It should probably be called "remove formatting", not "remove format".

**Review by tomaskikutis — APPROVED** [review]

File: scripts/apps/workspace/content/directives/ContentProfileSchemaEditor.ts (hunk 1 above)
Comment (pablopunk) [reply scripts/apps/workspace/content/directives/ContentProfileSchemaEditor.ts:94]: I thought I read it this way on the requirements but I can't see it anymore so I'll change it cause it makes more sense

Comment (pablopunk) [reply scripts/apps/workspace/content/directives/ContentProfileSchemaEditor.ts:94]: you mean "remove all formatting" or change the old one too?

Comment (tomaskikutis) [reply scripts/apps/workspace/content/directives/ContentProfileSchemaEditor.ts:94]: It'd be good to rename both since it's the same term.

Comment (pablopunk) [reply scripts/apps/workspace/content/directives/ContentProfileSchemaEditor.ts:94]: I think that would remove the option from current instances (as this is an id). Also, I think I'm gonna put all these into a type so we can keep track of them in other files

Comment (tomaskikutis) [reply scripts/apps/workspace/content/directives/ContentProfileSchemaEditor.ts:94]: You can keep the ID and only change naming in function/variable names.

**Pull request merged**